packages/attribute-registry-process/test/vitest.api.setup.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { vi } from "vitest";
+import { Request, Response, NextFunction } from "express";
+
+vi.mock("pagopa-interop-application-audit", async () => ({
+  applicationAuditBeginMiddleware: vi.fn(
+    async () => (_req: Request, _res: Response, next: NextFunction) => next()
+  ),
+  applicationAuditEndMiddleware: vi.fn(
+    async () => (_req: Request, _res: Response, next: NextFunction) => next()
+  ),
+}));
+
+vi.mock("pagopa-interop-commons", async () => {
+  const actual = await vi.importActual<typeof import("pagopa-interop-commons")>(
+    "pagopa-interop-commons"
+  );
+  return {
+    ...actual,
+    authenticationMiddleware: vi.fn(
+      () =>
+        async (
+          req: Request,
+          _res: Response,
+          next: NextFunction
+        ): Promise<unknown> => {
+          try {
+            const jwtToken = jwtFromAuthHeader(req, genericLogger);
+            const decoded = decodeJwtToken(jwtToken, genericLogger);
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            const ctx = (req as any).ctx || {};
```

Comment:
If the above doesn't work, you can also improve this to cast to a more precise type:
```suggestion
            const ctx = (req as Request & { ctx?: AppContext }).ctx || {};
```

Reply:
The above work perfectly 👍